wftests/warrior_tests/suites/framework_tests/ts_wrapper_with_debug_negative.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" ?>
```

Comment:
Execution Summary :
We have status for setup,cleanup and Testcase. Debug section status also need to be seen in the execution Summary.